.github/workflows/build.yml

```diff
@@ -101,17 +101,18 @@ jobs:
       - name: Copy bootloader_dio_40m.bin
         run: cp ~/.platformio/packages/framework-arduinoespressif32/tools/sdk/esp32/bin/bootloader_dio_40m.bin .pio/build/${{ matrix.environment }}/bootloader_dio_40m.bin
 
-      - uses: actions/upload-artifact@v3
-        with:
-          name: opendtu-${{ matrix.environment }}
-          path: |
-            .pio/build/${{ matrix.environment }}/opendtu-${{ matrix.environment }}.bin
-            .pio/build/${{ matrix.environment }}/partitions.bin
-            .pio/build/${{ matrix.environment }}/bootloader_dio_40m.bin
-            .pio/build/${{ matrix.environment }}/boot_app0.bin
+      - name: Generate Tag name for release
+        run: |
+          echo "tag=build-$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
```

Comment:
Something like 
```
echo "tag=build-$(git rev-parse --short HEAD)-$(date -I)" >> $GITHUB_OUTPUT
```

Reply:
> Great idea, but I would add an ISO timestamp to the tag for readability and for the less experienced users.

I like that! I just used the plain Commit-ID because currently the `Info -> System` page shows it as the "Firmware version".

![Screenshot of Info -> System](https://user-images.githubusercontent.com/54552094/219948128-64bb190c-a222-49a9-a3cf-83d648733a7b.png)

For even more improved readability, I would swap these values. So date first, then commit id. Example:

> build-2023-02-19-da86714